commerce/endpoint/account/address/classes/AddressExtensionProviderSample.cls

```diff
@@ -0,0 +1,19 @@
+// This corresponds to /commerce/webstores/${webstoreId}/accounts/${accountId}/addresses/${addressId} and Commerce_Endpoint_Account_Address
```

Comment:
@pdisabito Do we need to include copyright headers for sample code?

Reply:
@dbarbato-sf and @jnadeausf -- This is a legal question, and I'm not sure of the answer.

I checked the following repo to see what it does:

https://github.com/trailheadapps/lwc-recipes

This repo contains a [LICENSE.md](https://github.com/trailheadapps/lwc-recipes/blob/main/LICENSE.md) file for "Creative Commons Legal Code."

Not sure if we want to follow this pattern, but if we do, I think it's okay to omit the copyrights. Most of the sample files in this repo do not have a copyright.

Reply:
Checked another repo, which does include copyrights:

https://github.com/forcedotcom/commerce-on-lightning/blob/develop/LICENSE.txt

Not sure why legal requrements are different.